src/proxy/logging/LogAccess.cc

```diff
@@ -2096,6 +2096,27 @@ LogAccess::marshal_client_req_squid_len(char *buf)
   return INK_MIN_ALIGN;
 }
 
+/*-------------------------------------------------------------------------
+  Client request squid length plus TLS handshake bytes received for TLS connections.
+  -------------------------------------------------------------------------*/
+int
+LogAccess::marshal_client_req_squid_len_tls(char *buf)
+{
+  if (buf) {
+    int64_t val = 0;
+
+    if (m_client_request) {
+      val = m_client_request->length_get() + m_http_sm->client_request_body_bytes;
```

Comment:
What if the handshake has early data? I guess the size of the request gets double counted.